Procfile.dev

```diff
@@ -0,0 +1,2 @@
+web: env RUBY_DEBUG_OPEN=true bin/rails server
+admin_tailwind: cd admin && bundle exec rake solidus_admin:tailwind:watch
```

Comment:
I think this can be replaced by `bin/rails solidus_admin:tailwind:watch`

Reply:
Using `bin/rails` here won't find the task. I think that's because it is not defined within `lib/tasks`. Instead, it's placed directly in the `Rakefile`. I did that purposefully because that task shouldn't be exposed to the host application. They don't need to compile a file that we use for development. Let me know if you think otherwise :pray: